### ISSUE TITLE
Clearing Shields Per-Site Settings

### ISSUE THREAD

**Issue by tildelowengrimm** [brave/brave-browser#3926]
None of the options in the "Clear Browsing Data" settings item explicitly call out Shields, but certainly some of them affect Shields, because I just restarted my browser with some of them selected and my per-site and global Shields settings were reset.

We should specifically call out Shields settings in the clear-on-exit menu.

**Comment by sbtnc** [comment]
Yes, indeed. When "Site Settings" is checked in "Clear browsing data", it causes a reset on some (not all) shields settings.

Maybe "Site Settings" label is misleading (as I would expect specifics settings not global ones) or it's a bug.

For my case, I have a privacy oriented profile for daily surfing (whitelisting approach). Thus clearing all browsing data on exit does make sense. However, defaults shields settings get reseted on restart which are least private settings.

## Steps to Reproduce

1. In "Settings" > "Brave shields default" set to non defaults "Ad Control", "Cookie Control" and "Fingerprinting protection";
2. In "Settings" > "Clear browsing data" > "On exit" check "Site Settings";
3. Restart Brave Browser;

## Actual result:
Global settings are back to defaults.

## Expected result:
Global settings don't get back to defaults.

## Reproduces how often: 
Always.

## Brave version
Brave | 0.64.77 Chromium: 74.0.3729.169 (Official Build) (64-bit)
-- | --
Revision | 78e4f8db3ce38f6c26cf56eed7ae9b331fc67ada-refs/branch-heads/3729@{#1013}
OS | Mac OS X

**Comment by rebron** [comment]
@tomlowenthal Fixed with #8826?

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
